zipline-loader/build.gradle.kts

```diff
@@ -12,6 +12,7 @@ plugins {
   id("org.jetbrains.dokka")
   id("com.vanniktech.maven.publish.base")
   id("com.squareup.sqldelight")
+  id("de.undercouch.download") version "5.3.0"
```

Comment:
Weird. How was the build even working without this before?

In general we prefer the versions to be centrally defined. This should be put into the `gradle/libs.gradle.toml` file and then applied in the root `build.gradle.kts` using the old `dependencies`+`classpath` mechanism. That will make the plugin available without needing an explicit `id` here.

Reply:
They added a workaround to make it internal in 1.7.20 https://github.com/JetBrains/kotlin/commit/b6edf9a86eaf00096b41dd1a69fb78e1af9b03b5

Reply:
Ah, I didn't realize we were implicitly getting it from the Kotlin Gradle plugin. Glad they fixed it.